daemon/logs.go

```diff
@@ -58,7 +55,21 @@ func (daemon *Daemon) ContainerLogs(container *Container, config *ContainerLogsC
 		tailLines = -1
 	}
 
+	outStream := config.OutStream
+	errStream := outStream
+
+	// write an empty chunk of data (this is to ensure that the
+	// HTTP Response is sent immediately, even if the container has
+	// not yet produced any data)
+	outStream.Write(nil)
+
 	logrus.Debug("logs: begin stream")
+	if !container.Config.Tty {
+		errStream = stdcopy.NewStdWriter(outStream, stdcopy.Stderr)
+		outStream = stdcopy.NewStdWriter(outStream, stdcopy.Stdout)
+	}
+	config.OutStream = outStream
```

Comment:
btw I don't see where `config.OutStream` value used below. Maybe we don't need to modify config here?